cmd/titus-vpc-tool/setup_container.go

```diff
@@ -13,26 +13,33 @@ import (
 )
 
 func setupContainercommand(ctx context.Context, v *pkgviper.Viper, iipGetter instanceIdentityProviderGetter) *cobra.Command {
+	withTransition := false
+
+	netNS := []interface{}{v.GetInt("netns")}
+	if v.GetString("trans-netns") != "" {
+		withTransition = true
+		netNS = append(netNS, v.GetString("trans-netns"))
+	}
 	cmd := &cobra.Command{
 		Use:   "setup-container",
 		Short: "Setup networking for a particular container",
 		RunE: func(cmd *cobra.Command, args []string) error {
-			netns := v.GetInt("netns")
 			bandwidth := v.GetInt64("bandwidth")
 			burst := v.GetBool("burst")
 			jumbo := v.GetBool("jumbo")
 			switch strings.ToLower(v.GetString(generationFlagName)) {
 			case "v1":
-				return container.SetupContainer(ctx, iipGetter(), netns, uint64(bandwidth), burst, jumbo)
+				return container.SetupContainer(ctx, iipGetter(), netNS, withTransition, uint64(bandwidth), burst, jumbo)
 			case "v2", "v3":
-				return container2.SetupContainer(ctx, iipGetter(), netns, uint64(bandwidth), burst)
+				return container2.SetupContainer(ctx, iipGetter(), netNS, withTransition, uint64(bandwidth), burst)
 			default:
 				return fmt.Errorf("Version %q not recognized", v.GetString(generationFlagName))
 			}
 		},
 	}
 
 	cmd.Flags().Int("netns", 3, "The File Descriptor # of the network namespace to setup")
+	cmd.Flags().String("transition-netns", "trans-netns", "The name # of the network namespace to setup for transition")
```

Comment:
I think the fact that we have an fd above (`3`) and a second thing which is a string which is the name of a namespace, we are setting ourselves up for confusion.

Do we even need it to be a name? Can it be a bool ?